src/eos-metrics-instrumentation.c

```diff
@@ -946,6 +971,33 @@ network_dbus_proxy_new (void)
 }
 
 static gboolean
+record_windows_licenses (gpointer unused)
+{
+    g_autoptr(GFile) tables = g_file_new_for_path (ACPI_TABLES_PATH);
+    g_auto(GVariantBuilder) builder;
+    gsize i;
+
+    g_variant_builder_init (&builder, G_VARIANT_TYPE ("a{sb}"));
```

Comment:
Although we would have  many of these strings a dictionary like {"MSDM": true, "SLIC": false} would be much feasible to data module as the rest of the dictionary like events